### ISSUE TITLE
Display package test fails on local Windows build

### ISSUE THREAD

**Issue by patzielinski** [gittuf/gittuf#502]
### What happened?

Building gittuf on a Windows machine in PowerShell or the basic terminal (`cmd.exe`) causes the test for the display package to fail. Switching to the git bash (i.e. using MinGW) makes this package test pass, but the ssh-keygen test fails instead.

### Relevant log output

```Shell
--- FAIL: TestNewDisplayWriter (0.02s)
    --- FAIL: TestNewDisplayWriter/simple_test_with_paging (0.02s)
        display_test.go:51: unexpected result with Display(), got stdout = Hello, world!
            , want Hello, world!
FAIL
FAIL    github.com/gittuf/gittuf/internal/display       1.332s
FAIL
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by adityasaky** [comment]
looks like it's a newline mismatch, yes?

**Comment by adityasaky** [comment]
Fixed by #507
